build/docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
         release: "2022.2.1"
     depends_on:
       - base-notebook
-    image: daskdev/dask-notebook:latest
+    image: ghcr.io/dask-notebook:latest
```

Comment:
Here is an example of an incorrect image reference, it should be `ghcr.io/dask/dask-notebook:latest`